Auth0/OAuth2Grant.swift

```diff
@@ -154,20 +154,20 @@ struct PKCE: OAuth2Grant {
             let string = "No code found in parameters \(values)"
             return callback(.failure(AuthenticationError(string: string)))
         }
-        let idToken = values["id_token"]
+        let isFrontChannelIdTokenExpected = responseType.contains(.idToken)
+        let frontChannelIdToken = values["id_token"]
         let responseType = self.responseType
         let authentication = self.authentication
         let verifier = self.verifier
         let redirectUrlString = self.redirectURL.absoluteString
         let clientId = authentication.clientId
-        let isFrontChannelIdTokenExpected = responseType.contains(.idToken)
         let validatorContext = IDTokenValidatorContext(authentication: authentication,
                                                        issuer: self.issuer,
                                                        leeway: self.leeway,
                                                        maxAge: self.maxAge,
                                                        nonce: self.defaults["nonce"],
                                                        organization: self.organization)
-        validateFrontChannelIDToken(idToken: idToken, for: responseType, with: validatorContext) { error in
+        validateFrontChannelIDToken(idToken: frontChannelIdToken, for: responseType, with: validatorContext) { error in
```

Comment:
This method (and everything related to front-channel ID Tokens) will be removed in a future PR, as we'll only support `code`.